pkg/resources/network_policy_acceptance_test.go

```diff
@@ -0,0 +1,61 @@
+package resources_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccNetworkPolicyAttachment(t *testing.T) {
+	if _, ok := os.LookupEnv("SKIP_NETWORK_POLICY_TESTS"); ok {
+		t.Skip("Skipping TestAccNetworkPolicyAttachment")
+	}
+
+	policyName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+
+	resource.Test(t, resource.TestCase{
+		Providers: providers(),
+		Steps: []resource.TestStep{
+			{
+				Config: networkPolicyAttachmentConfig(policyName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_network_policy_attachment.test", "network_policy_name", policyName),
+					resource.TestCheckResourceAttr("snowflake_network_policy_attachment.test", "set_for_account", "false"),
+					resource.TestCheckResourceAttr("snowflake_network_policy_attachment.test", "users.#", "2"),
+				),
+			},
+			// IMPORT
+			{
+				ResourceName:      "snowflake_network_policy_attachment.test",
+				ImportState:       true,
+				ImportStateVerify: false,
```

Comment:
Is there a reason for this to be false?

Reply:
I set this to false while developing because I wasn't able to get it to work, I thought due to the inability to _actually_ implement `ReadNetworkPolicyAttachment`

But it looks like it works now so switching it back to true

Reply:
annnd I was wrong -- it's actually still not working:
```
=== RUN   TestAccNetworkPolicyAttachment
    TestAccNetworkPolicyAttachment: testing.go:683: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.

        (map[string]string) {
        }


        (map[string]string) (len=5) {
         (string) (len=19) "network_policy_name": (string) (len=10) "uowauijhoy",
         (string) (len=15) "set_for_account": (string) (len=5) "false",
         (string) (len=7) "users.#": (string) (len=1) "2",
         (string) (len=16) "users.2494182523": (string) (len=10) "test-user1",
         (string) (len=16) "users.3213322168": (string) (len=10) "test-user2"
        }

--- FAIL: TestAccNetworkPolicyAttachment (7.03s)
```

I think because `ReadNetworkPolicyAttachment` isn't really implemented?

Reply:
Just confirming you saw this last comment/[commit](https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/271/commits/53a3f9a1e452f6f78245cb9de1282aec69ef890b) @ryanking? Does the merge mean you're fine skipping `ImportStateVerify` given Snowflake limitations that prevent us from show/desc'ing anything about a policy attachment on users/account?

Reply:
Yeah its fine.

Reply:
Awesome, thanks for all the help getting this merged!